Week3/homework/1-step3.js

```diff
@@ -1,11 +1,12 @@
 'use strict';
 
 function foo(func) {
-  // What to do here? 
+  setTimeout(function () { bar() }, 5000);
```

Comment:
I am not pretty sure about what are you trying to do here, in all cases this will work fine after 5 seconds, but what if you have `bar` function in another file you can't pass it like so!

the only solution for this is pass it by arguments.